src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js

```diff
@@ -55,7 +55,9 @@ const ExchangeAccountAdd = ({ demo }) => {
   const testNet = watch("testNet");
   // Show testnet only for binance demo futures
   const showTestnet =
-    demo && exchangeType === "futures" && exchangeName.toLowerCase() === "binance";
+    demo &&
+    exchangeType === "futures" &&
+    (exchangeName.toLowerCase() === "binance" || exchangeName.toLowerCase() === "bitmex");
```

Comment:
`["binance", "bitmex"].includes(exchangeName.toLowerCase())` might be shorter but not a big deal